packages/ui/src/routes/bridge/BridgeSetupPage.tsx

```diff
@@ -556,6 +561,7 @@ const BridgeSetupPage: FunctionComponent<{}> = () => {
                         >
                             <input
                                 {...register("amount")}
```

Comment:
The `register` function already returns a ref and we're overriding it.

We can either use the `useMergeRef` hook or do this: 

```javascript
const { ref: amountInputRef, ...amountInputProps } = register("amount");

return (
<>
  ...
 <input  ref={amountInputRef} {...amountInputProps} id="amount" .../>
</>
)
```

Reply:
Good catch!

Reply:
Good catch!